Source/Response.swift

```diff
@@ -383,18 +388,21 @@ extension DownloadResponse: CustomStringConvertible, CustomDebugStringConvertibl
     /// The debug textual representation used when written to an output stream, which includes the URL request, the URL
     /// response, the temporary and destination URLs, the resume data, the response serialization result and the
     /// timeline.
-    public var debugDescription: String {
-        var output: [String] = []
-
-        output.append(request != nil ? "[Request]: \(request!.httpMethod ?? "GET") \(request!)" : "[Request]: nil")
-        output.append(response != nil ? "[Response]: \(response!)" : "[Response]: nil")
-        output.append("[TemporaryURL]: \(temporaryURL?.path ?? "nil")")
-        output.append("[DestinationURL]: \(destinationURL?.path ?? "nil")")
-        output.append("[ResumeData]: \(resumeData?.count ?? 0) bytes")
-        output.append("[Result]: \(result.debugDescription)")
-        output.append("[Timeline]: \(timeline.debugDescription)")
-
-        return output.joined(separator: "\n")
+    public var debugDescription: String {     
+        let requestDescription = request.map { "\($0.httpMethod ?? "GET") \($0)"} ?? "nil"
+        let requestBody = request?.httpBody.map { String(decoding: $0, as: UTF8.self) } ?? "None"
+        let responseDescriotion = response.map { "\($0)" } ?? "nil"
```

Comment:
You've misspelled `Description` here. Please correct it.

Reply:
I've fixed it.